[Bug] [2.16alpha54] Crash on opening `Deck options`

Thank you always, developers.

###### Reproduction Steps

1. Update AnkiDroid to version 2.16alpha54 (the latest version)
2. Open AnkiDroid.
3. Long tap a deck on `Decks` screen.

4. Tap `Deck options` on the deck's popup menu.
![image](https://user-images.githubusercontent.com/10436072/161418898-5c9fbb85-e2a5-459b-8212-9c9d2f001e1d.png)

###### Expected Result
`Deck options` screen is shown, as in 2.16alpha53, in which a crash bug on opening `Deck options` (#10584) has already been fixed.
![image](https://user-images.githubusercontent.com/10436072/161419137-7a9b4ffd-f5e9-4d76-a60e-74c989295a97.png)

https://user-images.githubusercontent.com/10436072/161418984-5c7fcc96-8590-4865-a66f-723ad10dd08b.mp4

###### Actual Result

AnkiDroid crashes.

![image](https://user-images.githubusercontent.com/10436072/161421393-1fbaf7a1-35ae-409a-9f47-aca1dbab5a24.png)


https://user-images.githubusercontent.com/10436072/161421401-4721ee47-5d1f-4270-9e28-bdf0dc2a8fb2.mp4



###### Debug info
```
AnkiDroid Version = 2.16alpha54

Android Version = 9

Manufacturer = OPPO

Model = CPH1983

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 9; CPH1983 Build/PKQ1.190519.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/99.0.4844.73 Mobile Safari/537.36

ACRA UUID = 6ecf0de3-2b4c-4450-89b6-dde5a2b6d317

Scheduler = std2

Crash Reports Enabled = true

DatabaseV2 Enabled = true

```

###### Research
*Enter an [x] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

I've got the same while testing other things
```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.ichi2.anki, PID: 22266
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.ichi2.anki/com.ichi2.anki.DeckOptions}: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter defaultValue
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3846)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4022)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2336)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:246)
        at android.app.ActivityThread.main(ActivityThread.java:8653)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
     Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter defaultValue
        at com.ichi2.ui.SeekBarPreference.onSetInitialValue(Unknown Source:6)
        at android.preference.Preference.dispatchSetInitialValue(Preference.java:1738)
        at android.preference.Preference.onAttachedToHierarchy(Preference.java:1505)
        at android.preference.PreferenceGroup.addPreference(PreferenceGroup.java:185)
        at android.preference.PreferenceGroup.addItemFromInflater(PreferenceGroup.java:127)
        at android.preference.PreferenceGroup.addItemFromInflater(PreferenceGroup.java:52)
        at android.preference.GenericInflater.rInflate(GenericInflater.java:497)
        at android.preference.GenericInflater.rInflate(GenericInflater.java:502)
        at android.preference.GenericInflater.rInflate(GenericInflater.java:502)
        at android.preference.GenericInflater.inflate(GenericInflater.java:334)
        at android.preference.GenericInflater.inflate(GenericInflater.java:271)
        at android.preference.PreferenceManager.inflateFromResource(PreferenceManager.java:343)
        at android.preference.PreferenceActivity.addPreferencesFromResource(PreferenceActivity.java:1909)
        at com.ichi2.anki.DeckOptions.onCreate(DeckOptions.java:676)
        at android.app.Activity.performCreate(Activity.java:8207)
        at android.app.Activity.performCreate(Activity.java:8191)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3819)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4022) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2336) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:246) 
        at android.app.ActivityThread.main(ActivityThread.java:8653) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130) 
D/UsageAnalytics: sendAnalyticsException() description/fatal: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter defaultValue/true
I/ACRA: ACRA is disabled for com.ichi2.anki - forwarding uncaught Exception on to default ExceptionHandler
```

--- a/AnkiDroid/src/main/java/com/ichi2/anki/DeckOptions.java
+++ b/AnkiDroid/src/main/java/com/ichi2/anki/DeckOptions.java
@@ -39,6 +39,7 @@ import com.ichi2.anim.ActivityTransitionAnimation;
 import com.ichi2.anki.exception.ConfirmModSchemaException;
 import com.ichi2.anki.receiver.SdCardReceiver;
 import com.ichi2.anki.services.ReminderService;
+import com.ichi2.annotations.NeedsTest;
 import com.ichi2.async.CollectionTask;
 import com.ichi2.async.TaskListenerWithContext;
 import com.ichi2.async.TaskManager;
@@ -80,6 +81,7 @@ import static com.ichi2.anim.ActivityTransitionAnimation.Direction.FADE;
 /**
  * Preferences for the current deck.
  */
+@NeedsTest("onCreate - to be done after preference migration (5019)")
 public class DeckOptions extends AppCompatPreferenceActivity implements OnSharedPreferenceChangeListener {
 
     private DeckConfig mOptions;

--- a/AnkiDroid/src/main/java/com/ichi2/ui/SeekBarPreference.kt
+++ b/AnkiDroid/src/main/java/com/ichi2/ui/SeekBarPreference.kt
@@ -102,7 +102,7 @@ class SeekBarPreference(context: Context, attrs: AttributeSet) : android.prefere
         mSeekBar!!.progress = (mValue - mMin) / mInterval
     }
 
-    override fun onSetInitialValue(restore: Boolean, defaultValue: Any) {
+    override fun onSetInitialValue(restore: Boolean, defaultValue: Any?) {
         super.onSetInitialValue(restore, defaultValue)
         mValue = getPersistedInt(mDefault)
         mValue = if (restore) {

